modules/node_modules/@ncigdc/containers/CaseTr.js

```diff
@@ -35,7 +36,17 @@ export type TProps = {|
   theme: Object,
 |};
 
-export const CaseTrComponent = ({ node, index, theme }: TProps) => {
+function massageRelayFiles(files: Array): Array {
+  return _.get(files, 'hits.edges', [])
```

Comment:
Well that style of null checking is used all over this repo already instead of _.get(), so I'm pretty used to not using _.get() haha. But I've already banged my head over the jest setup way too much so if you wanna help in this PR https://github.com/NCI-GDC/portal-ui/pull/1268, if _.uniq is called it'll try to call lodash2 instead of 4 and die.